<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountDLG</class>
 <widget class="QDialog" name="AccountDLG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>账户管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>账户</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="TaskTableWidget" name="tWAccounts">
            <column>
             <property name="text">
              <string>AppleID</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>密码</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>身份证号码</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>手机号码</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="gBTaskinfo">
       <property name="title">
        <string>添加账户</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>账户ID:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lEAccount"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>密码：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lEPasswd">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>身份证号：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lEGovId"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>手机号：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lEPhoneNumber"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pBAdd">
            <property name="text">
             <string>添加</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBImportfromfile">
            <property name="text">
             <string>从文件导入</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBClear">
            <property name="text">
             <string>清空</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBOk">
            <property name="text">
             <string>确定</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pBCancel">
            <property name="text">
             <string>取消</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaskTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">tasktablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pBAdd</sender>
   <signal>clicked()</signal>
   <receiver>AccountDLG</receiver>
   <slot>addAccount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClear</sender>
   <signal>clicked()</signal>
   <receiver>AccountDLG</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBOk</sender>
   <signal>clicked()</signal>
   <receiver>AccountDLG</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBCancel</sender>
   <signal>clicked()</signal>
   <receiver>AccountDLG</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBImportfromfile</sender>
   <signal>clicked()</signal>
   <receiver>AccountDLG</receiver>
   <slot>importFromFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addAccount()</slot>
  <slot>clear()</slot>
  <slot>randomGovernId()</slot>
  <slot>importFromFile()</slot>
 </slots>
</ui>
